dotnet/src/Connectors/Connectors.AI.HuggingFace/TextCompletion/HuggingFaceTextCompletion.cs

```diff
@@ -71,7 +71,7 @@ public HuggingFaceTextCompletion(string model, string? apiKey = null, HttpClient
     {
         foreach (var completion in await this.ExecuteGetCompletionsAsync(text, cancellationToken).ConfigureAwait(false))
         {
-            yield return completion;
+            yield return (ITextStreamingResult)completion;
```

Comment:
nit: Does it mean that streaming is not supported here, and we just return completed result? I'm wondering if it makes sense to throw the exception instead to show end users that streaming capability is not supported in this connector (at least for now).